### PR TITLE
[MM-41998] Channel Info RHS: display number of files for a channel

### PR DESCRIPTION
**Pull request by JulienTant**
#### Summary
This PR adds the number of files in a channel in the channel stats.

QA will be done on the frontend part.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-41998

#### Webapp PR

https://github.com/mattermost/mattermost-webapp/pull/10045

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Review by neallred — APPROVED** [review]
LGTM!

**Review by pvev — APPROVED** [review]
LGTM! :shipit:

File: store/sqlstore/channel_store.go [hunk 1]
```diff
@@ -2188,6 +2188,26 @@ func (s SqlChannelStore) GetMemberCountFromCache(channelId string) int64 {
 	return count
 }
 
+//nolint:unparam
```
[the remark this answers is not in the document]
Comment (JulienTant) [reply store/sqlstore/channel_store.go:2191]: Result of a lazy CopyPasta 🤦🏻 good catch!

**Pull request merged**